infrastructure/templates/public-api/components/alerts/dynamicMetricAlert.bicep

```diff
@@ -25,11 +25,11 @@ param windowSize WindowSize = 'PT5M'
 @description('The alert severity.')
 param severity Severity = 'Error'
 
-param sensitivity Sensitivity = 'High'
+param sensitivity Sensitivity = 'Low'
 
-param minFailingPeriodsToAlert int = 1
+param minFailingPeriodsToAlert int = 3
 
-param numberOfEvaluationPeriods int = 1
+param numberOfEvaluationPeriods int = 3
```

Comment:
Minor, but shouldn't these changes be in the EES-5686 PR given we introduce `dynamicMetricAlert.bicep` there? 

Doesn't make a lot of sense to have a follow-up PR that immediately changes the previous one.

Reply:
Yup, fair point - I have made these changes in the previous PR thanks.